dockerfiles/Grunt

```diff
@@ -0,0 +1,38 @@
+FROM envygeeks/alpine
+MAINTAINER "Brady Vitrano <brady.vitrano@rackspace.com>"
+COPY copy/ /
+RUN apk --update add build-base nasm autoconf libtool automake \
+                     ruby ruby-dev ruby-io-console ruby-irb ruby-json ruby-rake ruby-rdoc \
+                     nodejs \
+                     python \
+                     readline readline-dev libxml2 libxml2-dev libxslt  \
+                     libxslt-dev python zlib zlib-dev \
+                     libffi libffi-dev \
+                     git \
+                     && \
+
+    mkdir -p /home/bob && \
+    addgroup -Sg 1000 bob &&  \
+    adduser  -SG bob -u 1000 -h /home/bob bob && \
+    chown bob:bob /home/bob && \
+
+    cd ~ && \
+    yes | gem update --system --no-document -- --use-system-libraries && \
+    yes | gem update --no-document -- --use-system-libraries && \
+
+    cd /home/bob && \
+    gem install sass --no-document && \
+
```

Comment:
Add the following for firewall/proxy connections 

`git config --global url."https://".insteadOf git://`